application/logging_default_config.yaml

```diff
@@ -0,0 +1,34 @@
+version: 1
+disable_existing_loggers: False
+
+formatters:
+    default_formatter:
+        format: '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+        datefmt: '%Y-%m-%d %H:%M:%S'
+
+handlers:
+    console:
+        class: logging.StreamHandler
+        level: DEBUG
+        formatter: default_formatter
+        stream: ext://sys.stdout
```

Comment:
I'm fine using stderr, which is unbuffered, during development. Feel free to object if there's a strong reason we should be using stdout.

```suggestion
```

Reply:
I didn't know the clear difference between stderr and stdout so by convention I just chose stdout, I'll change it.